Gemfile

```diff
@@ -6,6 +6,7 @@ source "https://rubygems.org"
 RAILS_VERSION = ENV.fetch("RAILS_VERSION", "6.0.0")
 
 gem "rails", "~> #{RAILS_VERSION}"
+gem "concurrent-ruby", "1.3.4"
```

Comment:
What's this for?

Reply:
All rails versions before 7.1 have a direct dependency on concurrent-ruby (We use an older version for testing), but concurrent-ruby versions 1.3.5+ removed the dependency on Logger. When using Rails 7.0- and concurrent-ruby 1.3.5+, you get `uninitialized constant ActiveSupport::LoggerThreadSafeLevel::Logger (NameError)` when starting tests-- so I just pinned the version to 1.3.4 instead of upgrading the dummy rails installation.

Reply:
Added a clarifying comment to the Gemfile. This is not a gem dependency, just a developer one